examples/backtest.py

```diff
@@ -3,8 +3,8 @@
 
 class BackTest(BaseAlgo):
     
-    def main(self, _):
```

Comment:
The new `main()` function interface as of today only takes the `self` parameter. This should be applied to all `main()` functions in documentations and examples.